sjb/config/common/test_cases/origin_release_install_azure_39.yml

```diff
@@ -163,6 +163,34 @@ extensions:
           -e "openshift_azure_input_image_prefix=centos7" \
           playbooks/azure/openshift-cluster/launch.yml
 
+    - type: "script"
+      title: "run the acs-engine e2e test"
+      script: |-
+        git clone -b custom-images-e2e https://github.com/kargakis/acs-engine.git
+        cd acs-engine
+
+        set -a +o xtrace
+        . /data/src/github.com/openshift/release/cluster/test-deploy/azure/credentials
+        set +a -o xtrace
+
+        IMAGE_NAME=$(TYPE=azure /data/src/github.com/openshift/release/cluster/bin/ansible.sh az image list -g ${RESOURCE_GROUP} --query "[?starts_with(name, 'centos7-') && tags.valid=='true'] | sort_by(@, &name) | [-1]")
```

Comment:
Oh dear.  At this stage the image we're working on should not be marked as valid yet.
- build_node_image.yml shouldn't tag valid=true - create a completely new although very simple playbook to do that
- build_node_image.yml needs somehow to output the name of the trial image it built
- that needs to be persisted
- launch.yml needs to take in the image name as an override
- ditto here we need to pass in the image name into the docker container
- right at the end, if the test pass, then you should call the new playbook to mark the image as valid